packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass

```diff
@@ -47,13 +54,32 @@
   text-transform: uppercase
   white-space: nowrap
 
+.v-calendar-weekly__head-weeknumber
+  position: relative
+  border-top-left-radius: 4px
+  border-top-right-radius: 4px
+  flex: 0 0 24px
+  top: 0.2em
```

Comment:
what is this for? it causes the space at the top, is it expected?
![image](https://user-images.githubusercontent.com/15625235/80186378-2e240c80-860e-11ea-98bb-f1dd0d558652.png)

could be a sass var too, along with border radius